osquery/tables/system/tests/system_tables_tests.cpp

```diff
@@ -128,6 +128,36 @@ TEST_F(SystemsTablesTests, test_processes_memory_cpu) {
   EXPECT_GE(value - cpu_start, 0U);
 }
 
+TEST_F(SystemsTablesTests, test_processes_disk_io) {
+  // TODO: Remove once implemented on these platforms.
+  if (!isPlatform(PlatformType::TYPE_LINUX) &&
```

Comment:
Is this basically PlatformType::POSIX?

Reply:
No, because I don't have FreeBSD support.